README.md

```diff
@@ -949,7 +949,6 @@ Here is a list of template creators:
    * C# ASP.NET Core 3.0: @A-Joshi
    * C# APS.NET Core 3.1: @phatcher
    * C# Azure functions: @Abrhm7786
-   * C# NancyFX: @mstefaniuk
```

Comment:
I'll add it back after merging it into master as old versions still have this generator available.